src/Apps/Artwork/Components/ArtworkVideoPlayer/__tests__/ArtworkVideoPlayer.jest.tsx

```diff
@@ -1,33 +1,132 @@
-import React from "react"
-import { render } from "@testing-library/react"
-import {
-  ArtworkVideoPlayer,
-  ArtworkVideoPlayerProps,
-} from "../ArtworkVideoPlayer"
+import { act, fireEvent, waitFor } from "@testing-library/react"
+import { ArtworkVideoPlayer_Test_Query } from "__generated__/ArtworkVideoPlayer_Test_Query.graphql"
+import { ArtworkVideoPlayerFragmentContainer } from "Apps/Artwork/Components/ArtworkVideoPlayer/ArtworkVideoPlayer"
+import { useTracking } from "react-tracking"
+import { setupTestWrapperTL } from "DevTools/setupTestWrapper"
+import { graphql } from "react-relay"
+
+jest.unmock("react-relay")
+jest.mock("react-tracking")
 
 describe("ArtworkVideoPlayer", () => {
+  jest.useFakeTimers()
+
+  const mockUseTracking = useTracking as jest.Mock
   const image = { type: "Image" }
   const video = { type: "Video", url: "test-url", height: 360, width: 640 }
-  const defaultProps = figures =>
-    ({
-      activeIndex: 0,
-      artwork: {
-        figures,
-      },
-      maxHeight: 800,
-    } as ArtworkVideoPlayerProps)
+
+  const { renderWithRelay } = setupTestWrapperTL<ArtworkVideoPlayer_Test_Query>(
+    {
+      Component: props => (
+        <ArtworkVideoPlayerFragmentContainer
+          activeIndex={0}
+          artwork={props.artwork!}
+          maxHeight={500}
+        />
+      ),
+      query: graphql`
+        query ArtworkVideoPlayer_Test_Query {
+          artwork(id: "foo") {
+            ...ArtworkVideoPlayer_artwork
+          }
+        }
+      `,
+    }
+  )
+
+  beforeEach(() => {
+    mockUseTracking.mockReturnValue({ trackEvent: jest.fn() })
+  })
+
+  afterEach(() => {
+    jest.resetAllMocks()
+  })
 
   it("renders with video figure", () => {
-    const { container } = render(
-      <ArtworkVideoPlayer {...defaultProps([image, video])} />
```

Comment:
@laurabeth - if a component is a relay component we should always be using our relay test tools which auto generate fixture data for us